draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1312,7 +1312,7 @@ uCDN-A                         dCDN-B                         dCDN-C
          <xref target="sect-5.2.6"/>.  An empty array is allowed and is equivalent to
          omitting "errors" from the object.
 	<vspace blankLines="1"/>
-	Mandatory: No.
+	Mandatory: Yes for "eunsupported", "espec", or "esubject" errors. Otherwise, optional.
```

Comment:
should be mandatory if we have an error, but it is generally not mandatory.
Anyway, this definition came from the original RFC and I would prefer not to turn something to mandatory

Reply:
Good point on the original RFC.  I will change it back in the next commit.